src/window_equipstatus.cpp

```diff
@@ -121,12 +121,17 @@ void Window_EquipStatus::DrawParameter(int cx, int cy, int type) {
 	contents->TextDraw(cx, cy, Font::ColorDefault, std::to_string(value), Text::AlignRight);
 
 	if (draw_params) {
-		if (Player::IsCP932()) {
-			// Draw fullwidth arrow
-			contents->TextDraw(cx, cy, 1, "→");
+		if (lcf::Data::terms.easyrpg_equipment_arrow == lcf::Data::terms.kDefaultTerm) {
+			if (Player::IsCP932()) {
+				// Draw fullwidth arrow
+				contents->TextDraw(cx, cy, 1, "→");
+			} else {
+				// Draw arrow (+3 for center between the two numbers)
+				contents->TextDraw(cx + 3, cy, 1, ">");
+			}
 		} else {
-			// Draw arrow (+3 for center between the two numbers)
-			contents->TextDraw(cx + 3, cy, 1, ">");
+			// Draw arrow
+			contents->TextDraw(cx, cy, 1, lcf::Data::terms.easyrpg_equipment_arrow);
```

Comment:
You could use Font::Default().GetSize() to determine the width of the arrow and when it is < 12 (12 is the size of a full width glyph) you could center it.